modules/distribution/resources/migration-1.8.0_to_1.9.0/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/util/ResourceUtil.java

```diff
@@ -0,0 +1,278 @@
+                //pick from 16to17Migration/sql-scripts
+                resourcePath = "/16to17Migration/sql-scripts/";
+            } else {
+                throw new APIMigrationException("No query picked up for the given migrate version. Please check the migrate version.");
+            }
+
+            if(Constants.CONSTRAINT.equals(queryType)) {
+                resourcePath = CarbonUtils.getCarbonHome() + "/repository/components/dropins/";
+                //inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + "query.sql");
+                queryTobeExecuted = IOUtils.toString(new FileInputStream(new File(resourcePath + "query.txt")), "UTF-8");
+            } else {
+                inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + databaseType + ".sql");
+                queryTobeExecuted = IOUtils.toString(inputStream);
+            }
+
+                /*if (databaseType.equalsIgnoreCase("MYSQL")) {
+                    inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + "mysql.sql");
+                } else if (databaseType.equalsIgnoreCase("MSSQL")) {
+                    inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + "mssql.sql");
+                } else if (databaseType.equalsIgnoreCase("H2")) {
+                    inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + "h2.sql");
+                } else if (databaseType.equalsIgnoreCase("ORACLE")) {
+                    inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + "oracle.sql");
+                } else {
+                    inputStream = ResourceUtil.class.getResourceAsStream(resourcePath + "postgresql.sql");
+                }*/
+
+        } catch (IOException e) {
+            throw new APIMigrationException("Error occurred while accessing the sql from resources. " + e);
+        } catch (Exception e) {
+            throw new APIMigrationException("Error occurred while accessing the sql from resources. " + e);
+        } finally {
+            if (inputStream != null) {
+                inputStream.close();
+            }
+
+        }
+
+        return queryTobeExecuted;
+    }
+
+    /**
+     * To handle exceptions
+     *
+     * @param msg error message
+     * @throws APIMigrationException
+     */
+    public static void handleException(String msg, Throwable e) throws APIMigrationException {
+        log.error(msg, e);
+        throw new APIMigrationException(msg, e);
+    }
+
+    /**
+     * To copy a new sequence to existing ones
+     *
+     * @param sequenceDirectoryFilePath sequence directory
+     * @param sequenceName              sequence name
+     * @throws APIMigrationException
+     */
+    public static void copyNewSequenceToExistingSequences(String sequenceDirectoryFilePath, String sequenceName) throws APIMigrationException {
+        try {
+            String namespace = "http://ws.apache.org/ns/synapse";
```

Comment:
Can use a constant and define globally so that this assignment will not happen over and over again in the critical path.